docs/ASSEMBLY.md

```diff
@@ -14,47 +14,63 @@ Below are instructions for assembling your Sprig console.
 | 1        | LCD screen                                                                |
 | 1        | Raspberry Pi Pico                                                         |
 | 4        | clear backings (2 with D-pad cutouts, 2 solid)                            |
-| 1        | bag of 10 tactile button switches                                         |
-| 1        | bag of hardware for screen (2 spacers, 2 M2\*14mm screws, 2 hex nuts)\*    |
-| 1        | bag of hardware for backings (8 M2\*10mm screws, 8 hex nuts, allen key)\* |
+| 1        | bag of 10 tactile button switches (only 8 will be used)                   |
+| 1        | bag of hardware for screen (2 spacers, 2 M2\*14mm bolts, 2 hex nuts)\*    |
+| 1        | bag of hardware for backings (8 M2\*10mm bolts, 8 hex nuts, allen key)\*  |
 | 1        | adapter or micro-USB cable                                                |
 | 1        | bag of 2 AAA batteries                                                    |
 
-\* May include extra hardware.
+> [!NOTE]
+> Extra hardware may be included - it's all packed by hand. If you're missing a part, [DM `@graham` on the Hack Club Slack](https://hackclub.slack.com/team/U04QH1TTMBP) or email graham<img src="https://upload.wikimedia.org/wikipedia/commons/8/88/At_sign.svg" height=15px alt="@"></img>hackclub.com. 
 
 ## Instructions
 
-1. Connect the LCD screen to the pins on the top of the board.
+1. Slide the LCD screen into the socket on the top of the board.
 
 ![Photo of a PCB with LCD attached](https://cloud-6po09tv9d-hack-club-bot.vercel.app/2lcd-no-spacers.jpg)
 
-2. Add spacers, put the longer screws through the holes, and screw the nuts on the back side.
+> [!CAUTION]
+> Make sure that there are no pins floating out on either end! Misalignments can cause bad things to happen. 
+
+2. Slide one of the two plastic spacers (long tubes) under one of the two holes in the display. Slide one of the longer bolts through the screen, spacer, and circuit board and screw one of the nuts onto the other end. Use the allen key to tighten it. Repeat for the display's other hole.
```

Comment:
As with the previous comment, the spacer is like 2 connected tubes with a gap for the screen's SD card so it fits exactly.

Reply:
Please modify this copy to accommodate two possibilities - historically we have used 2 black cylindrical spacers, but we have transitioned to 1 3d-printed part that serves as both.